# Type: refactor
# Subtype: R10
# English Summary: Remove duplicate registration of the service
# Chinese Summary: 删除服务的重复注册
# Entry Number: 29

# === Git Diff Content ===

diff --git a/indexing-service/src/main/java/io/druid/indexing/common/index/EventReceiverFirehoseFactory.java b/indexing-service/src/main/java/io/druid/indexing/common/index/EventReceiverFirehoseFactory.java
index b9e420f..225d1b0 100644
--- a/indexing-service/src/main/java/io/druid/indexing/common/index/EventReceiverFirehoseFactory.java
+++ b/indexing-service/src/main/java/io/druid/indexing/common/index/EventReceiverFirehoseFactory.java
@@ -86,7 +86,6 @@ public class EventReceiverFirehoseFactory implements FirehoseFactory
     if (chatHandlerProvider.isPresent()) {
       log.info("Found chathandler of class[%s]", chatHandlerProvider.get().getClass().getName());
       chatHandlerProvider.get().register(serviceName, firehose);
-      chatHandlerProvider.get().register(serviceName.replaceAll(".*:", ""), firehose); // rofl
     } else {
       log.info("No chathandler detected");
     }
